src/combinatorics/simple_graph/connectivity.lean

```diff
@@ -1451,74 +1451,89 @@ def connected_component := quot G.reachable
 /-- Gives the connected component containing a particular vertex. -/
 def connected_component_mk (v : V) : G.connected_component := quot.mk G.reachable v
 
-@[simps] instance connected_component.inhabited [inhabited V] : inhabited G.connected_component :=
+namespace connected_component
```

Comment:
Putting everything in a namespace like this causes there to be a whole lot of changes not specific to the main content of the PR (and it makes it harder to merge PRs simultaneously). Please avoid doing this except in a refactor PR.

Reply:
Noted, thanks. Shall I revert that change?

Reply:
Or make a standalone PR that makes _only_ this change.

Reply:
Done!